minimint-bridge/Cargo.toml

```diff
@@ -11,9 +11,9 @@ crate-type = ["staticlib", "cdylib", "rlib"]
 anyhow = "1"
 futures = "0.3"
 
-mint-client = { git = "https://github.com/justinmoon/fedimint", branch = "fluttermint" }
-fedimint-api = { git = "https://github.com/justinmoon/fedimint", branch = "fluttermint" }
-fedimint-core = { git = "https://github.com/justinmoon/fedimint", branch = "fluttermint" }
+mint-client = { git = "https://github.com/justinmoon/fedimint", branch = "fluttermint-2" }
+fedimint-api = { git = "https://github.com/justinmoon/fedimint", branch = "fluttermint-2" }
+fedimint-core = { git = "https://github.com/justinmoon/fedimint", branch = "fluttermint-2" }
```

Comment:
can you put the commit in here as well

Reply:
As discussed -- no commit is fine because we're on a branch that only gets updates if they're helpful for fluttermint.